domains/rajveer.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "kyouhei14",
+        "email": "rajveerkobot@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://rajveer-is-dead.netlify.app"
```

Comment:
netlify is not supported yet (might be soon)

Reply:
also CNAME can't contain https:// or /

Reply:
> netlify is not supported yet (might be soon)

is there any website that host web pages for free?